includes/frontend.php

```diff
@@ -1339,6 +1338,8 @@ private function get_elementor_frontend_dependencies() {
 			);
 
 			$dependencies[] = 'swiper';
+			$dependencies[] = 'share-link';
```

Comment:
This script is used in the Pro without an explicit dependency (see the Share Buttons module). We have to solve it first.